fastlane/lib/fastlane/one_off.rb

```diff
@@ -17,14 +17,14 @@ def self.execute(args: nil)
         end
       end
 
-      raise "invalid syntax" unless action_name
+      UI.user_error!("invalid syntax") unless action_name
 
       class_name = action_name.fastlane_class + 'Action'
       class_ref = nil
       begin
         class_ref = Fastlane::Actions.const_get(class_name)
       rescue NameError
-        raise "Action not found"
+        UI.user_error!("Action '#{class_name}' not found")
```

Comment:
I wonder if this should be `crash!` instead? seems like you might want a trace of how you got here?